packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -326,7 +324,64 @@ describe('Select', () => {
         const selectedOption = pageObject.getSelectedOption();
         selectedOption!.textContent = 'foo';
         await waitForUpdatesAsync();
-        expect(element.displayValue).toBe('foo');
+        expect(pageObject.getDisplayText()).toBe('foo');
+
+        await disconnect();
+    });
+
+    it('after forcing Select to be blank, user can arrow down to first available option in dropdown', async () => {
```

Comment:
If you don't already have it, it would be good to have a similar test as this that leaves the select closed when pressing the down arrow key.